modules/jaeger-config-ingester.adoc

```diff
@@ -0,0 +1,92 @@
+////
+This REFERENCE module included in the following assemblies:
+-rhbjaeger-deploy.adoc
+////
+
+[id="jaeger-config-ingester_{context}"]
+= Jaeger Ingester configuration options
+
+Ingester is a service that reads from Kafka topic and writes to another storage backend (Elasticsearch).  If you are using the `allInOne` or `production` deployment strategies, you do not need to configure the Ingester service.
+
+
+.Jaeger default parameters
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+|spec:
+  strategy: streaming
+  ingester:
+    options: {}
+|Configuration options that define the Ingester service.
+|
+|
+
+|ingester:
+ options:
+  kafka:
+    consumer:
+      topic: jaeger-spans
+|The `topic` parameter identifies the Kafka configuration used by the collector to produce the messages, and the ingester to consume the messages.
+|Label for the consumer
+|
+
+|ingester:
+  options:
+    kafka:
+      consumer:
+        topic: jaeger-spans
+|Identifies the Kafka configuration used by the Ingester to consume the messages.
+|
+|
+
+|ingester:
+  options:
+    kafka:
+      consumer:
+        brokers: my-cluster-kafka-brokers.kafka:9092
+|Identifies the Kafka configuration used by the Ingester to consume the messages.
+|
+|
+
+|ingester:
+  options:
+    ingester:
+      deadlockInterval: 5
+| Specifies the interval (in seconds or minutes) that the Ingester should wait for a message before terminating.
+The deadlock interval is disabled by default (set to 0), to avoid the Ingester being terminated when no messages arrive while the system is being initialized.
+|Minutes and seconds, for example, 1m0s.
+|0
+|===
+
```

Comment:
Other ingester options that need to be documented are in: https://www.jaegertracing.io/docs/1.17/cli/#jaeger-ingester-elasticsearch (minus the `es` and `es-archive` and other exceptions as described in the collector section).